src/components/Login/LoginOTPCard.tsx

```diff
@@ -95,6 +102,22 @@ export const LoginOTPCard: FunctionComponent<LoginOTPCard> = ({
         alert(
           "Encountered an issue obtaining environment information. We've noted this down and are looking into it!"
         );
+      } else if (e instanceof LoginError) {
```

Comment:
Clarification: Are the errors being thrown from `validateOTP`?

Reply:
yup!